src/styles/game.css

```
@@ -0,0 +1,609 @@
+  left: 0;
+  right: 0;
+  bottom: 0;
+  box-shadow:
+    inset 0 0 120px rgba(0, 0, 0, 0.7),
+    inset 0 0 40px rgba(0, 0, 0, 0.4);
+  pointer-events: none;
+  z-index: 56;
+  animation: rgbBorder 6s infinite;
+}
+
+@keyframes rgbBorder {
```

Stylelint (17.2.0)

[error] Expected keyframe name "rgbBorder" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)